### PR TITLE
Question amendments 

### PR DESCRIPTION
**Pull request by cwrw**
- Combine both questions in to one to ask if the person requires medication
- If they require medication show the add another medication page, with multiple questions
- Amend the dependent question to have another option for people carrying the medication while moving
- Further question amendments in PER

### CODE REVIEW

File: frameworks/person-escort-record/questions/medication-time-last-given.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+type: text
+question: What time was it last given?
```
Comment (cwrw) [inline frameworks/person-escort-record/questions/medication-time-last-given.yml:2]: add hint text?

File: frameworks/person-escort-record/questions/medication-time-next-given.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+type: text
+question: What time should it next be given?
+description: Time next given
```
Comment (cwrw) [inline frameworks/person-escort-record/questions/medication-time-next-given.yml:3]: add hint text?

File: frameworks/person-escort-record/questions/medication-while-moving.yml [hunk 3]
```diff
@@ -1,14 +1,13 @@
...
 description: Medication while moving
 prefill: false
 options:
   -
     label: "Yes"
     value: "Yes"
     followup:
-      - medication-while-moving-details
-      - medication-carrier-while-moving
+      - prescribed-medication-carrier
```
Comment (cwrw) [inline frameworks/person-escort-record/questions/medication-while-moving.yml:10]: will dependent questions work with add another?

File: frameworks/person-escort-record/questions/history-of-self-harm-recency.yml [hunk 4]
```diff
@@ -1,24 +1,10 @@
-type: radio
-question: How recently did self-harm occur?
-description: Recency of self-harm
+type: text
+question: Date when the last incident took place
+hint: For example 28/10/2019 or 28 October 2019
```
Comment (cwrw) [inline frameworks/person-escort-record/questions/history-of-self-harm-recency.yml:3]: this was marked as tbd in ticket, let me know your thoughts

File: frameworks/person-escort-record/questions/medication-while-moving.yml [hunk 5]
```diff
@@ -1,14 +1,13 @@
 type: radio
-question: Do they need to take any medication while moving?
+question: Will the detainee need to take this medication while moving?
```
Comment (cwrw) [inline frameworks/person-escort-record/questions/medication-while-moving.yml:2]: change to person as we've not used detainee before

File: frameworks/person-escort-record/questions/medication-time-last-given.yml (hunk 1 above)
Comment (JadeNConroy) [reply frameworks/person-escort-record/questions/medication-time-last-given.yml:2]: I don't think hint text is needed

File: frameworks/person-escort-record/questions/medication-time-next-given.yml (hunk 2 above)
Comment (JadeNConroy) [reply frameworks/person-escort-record/questions/medication-time-next-given.yml:3]: I don't think hint text is needed

File: frameworks/person-escort-record/questions/history-of-self-harm-recency.yml (hunk 4 above)
Comment (JadeNConroy) [reply frameworks/person-escort-record/questions/history-of-self-harm-recency.yml:3]: Looks fine to me (if both date formats are acceptble).

File: frameworks/person-escort-record/questions/medication-while-moving.yml (hunk 5 above)
Comment (JadeNConroy) [reply frameworks/person-escort-record/questions/medication-while-moving.yml]: Could be 'Will they need to take this medication while moving'

File: frameworks/person-escort-record/questions/medication-time-next-given.yml (hunk 2 above)
Comment (Andy-Millar) [reply frameworks/person-escort-record/questions/medication-time-next-given.yml:3]: Agreed

File: frameworks/person-escort-record/questions/medication-while-moving.yml (hunk 5 above)
Comment (Andy-Millar) [reply frameworks/person-escort-record/questions/medication-while-moving.yml]: +1

File: README.md [hunk 6]
```diff
@@ -91,6 +91,8 @@ Question files should live in `questions/` and be stored in a flat structure. Fi
 - `display` - list of further display styles for question input
   - `rows` - the number of rows to display for `textarea` questions
   - `character_width` - the character width of a question input. Helps to make the input the right size for the content its intended for. Supported values: `20`, `10`, `5`, `4`, `3`, `2`.
+- `prefix` - text or characters to be used to before the field. Only applies to `text` fields.
```
Comment (cwrw) [inline README.md:94]:
```suggestion
- `prefix` - text or characters to be used before the field. Only applies to `text` fields.
```

File: README.md [hunk 7]
```diff
@@ -91,6 +91,8 @@ Question files should live in `questions/` and be stored in a flat structure. Fi
...
+- `suffix` - text or characters to be used to after the field. Only applies to `text` fields.
```
Comment (cwrw) [inline README.md:95]:
```suggestion
- `suffix` - text or characters to be used after the field. Only applies to `text` fields.
```

File: frameworks/person-escort-record/questions/property-bag-type.yml [hunk 8]
```diff
@@ -18,6 +18,7 @@ options:
       hint: For example, 23.99
       display:
         character_width: 5
+      prefix: £
```
Comment (cwrw) [inline frameworks/person-escort-record/questions/property-bag-type.yml:21]: this seems to be in favour of https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks/pull/20
which we can safely close now

**Review by teneightfive — APPROVED** [review]

**Pull request merged**